cluster/schema/schema.go

```diff
@@ -113,16 +113,16 @@ func (s *schema) ClassEqual(name string) string {
 }
 
 func (s *schema) unsafeClassEqual(name string) string {
-	for k := range s.classes {
-		if strings.EqualFold(k, name) {
-			return k
-		}
-	}
 	for alias := range s.aliases {
 		if strings.EqualFold(alias, name) {
 			return alias
 		}
 	}
+	for k := range s.classes {
+		if strings.EqualFold(k, name) {
```

Comment:
I know you copied this approach from existing approach. Looping through map to find the key is very inefficient and spoils the purpose of hash map. Why do we need EqualFold if we made sure class is Title case? can't we just `_, ok := s.classes[schema.UpperClassName(name)]`?